src/sentry/api/endpoints/project_keys.py

```diff
@@ -51,9 +51,8 @@ def get(self, request: Request, project) -> Response:
         """
         Return a list of client keys bound to a project.
         """
-        queryset = ProjectKey.objects.filter(
-            project=project, roles=F("roles").bitor(ProjectKey.roles.store)
-        )
+        objects = ProjectKey.objects if request.user.is_superuser else ProjectKey.objects.user()
```

Comment:
You should use `is_active_superuser` here to ensure that the active user has a superuser session active instead of only checking if they have the ability to become superuser.